api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java

```diff
@@ -0,0 +1,154 @@
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"partitions", "replicas", "config"})
+public class KafkaMirrorMakerSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private String whitelist;
+
+    private String blacklist;
+
+    private String source;
+
+    private KafkaMirrorMakerTls sourceTls;
+
+    private KafkaMirrorMakerAuthentication sourceAuthentication;
+
+    private String destination;
+
+    private KafkaMirrorMakerTls destinationTls;
+
+    private KafkaMirrorMakerAuthentication destinationAuthentication;
+
+    private Map<String, Object> config;
+
+    private Map<String, Object> additionalProperties;
+
+    @Description("The whitelist of the Mirror Maker")
+    public String getWhitelist() {
+        return whitelist;
+    }
+
+    public void setWhitelist(String whitelist) {
+        this.whitelist = whitelist;
+    }
+
+    @Description("The blacklist of the Mirror Maker")
+    public String getBlacklist() {
+        return blacklist;
+    }
+
+    public void setBlacklist(String blacklist) {
+        this.blacklist = blacklist;
+    }
+
+    @Description("Source TLS configuration")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public KafkaMirrorMakerTls getSourceTls() {
+        return sourceTls;
+    }
+
+    public void setSourceTls(KafkaMirrorMakerTls tls) {
+        this.sourceTls = tls;
+    }
+
+
+    @Description("Source TLS configuration")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public KafkaMirrorMakerTls getDestinationTls() {
+        return destinationTls;
+    }
+
+    public void setDestinationTls(KafkaMirrorMakerTls tls) {
+        this.destinationTls = tls;
+    }
+
+    @Description("Authentication configuration for Kafka Mirror Maker source")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public KafkaMirrorMakerAuthentication getSourceAuthentication() {
+        return sourceAuthentication;
+    }
+
+    public void setSourceAuthentication(KafkaMirrorMakerAuthentication authentication) {
+        this.sourceAuthentication = authentication;
+    }
+
+    @Description("Authentication configuration for Kafka Mirror Maker destination")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public KafkaMirrorMakerAuthentication getDestinationAuthentication() {
+        return destinationAuthentication;
+    }
+
+    public void setDestinationAuthentication(KafkaMirrorMakerAuthentication authentication) {
+        this.destinationAuthentication = authentication;
+    }
+
+    public void setDestination(String destination) {
+        this.destination = destination;
+    }
+
+    @Description("Destination address configuration")
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getDestination() {
+        return destination;
+    }
+
+
+    public void setSource(String source) {
+        this.source = source;
+    }
+
+    @Description("Destination address configuration")
```

Comment:
This one's the source not the destination. And what is expected here? Explain what an address configuration is supposed to be (is it a host:port?). Explain the syntax, give an example!